### PR TITLE
Chore/gdpr banner

### LINKED ISSUE #1645: GDPR - create notification banner w/link to updated privacy policy
Builds off of #1595

### PR DESCRIPTION
**Pull request by evwilkin**
Closes #1645 - this PR adds a banner to the bottom of the page for GDPR and links to privacy statement.  Banner is fixed to bottom of page, follows user as they navigate site until they close it out at which point it will not reappear for user.

### CODE REVIEW

File: packages/patternfly-4/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js [hunk 1]
```diff
@@ -244,6 +245,10 @@ export const SideNavLayout = ({
       {showBanner && (
         <Banner />
       )}
+      {console.log('sidenav/sidenav: ',showGdprBanner, 'showBanner: ', showBanner)}
```
Comment (redallen) [inline packages/patternfly-4/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js]: Can remove this line

File: packages/patternfly-4/gatsby-theme-patternfly-org/templates/mdx.js [hunk 2]
```diff
@@ -241,7 +241,9 @@ const MDXTemplate = ({ data, location, pageContext }) => {
         context={source}
         parityComponent={parityComponent}
         showBanner={showBanner}
+        showGdprBanner={showGdprBanner}
       >
+        {console.log('showGdprBanner: ', showGdprBanner, 'showBanner: ', showBanner)}
```
Comment (redallen) [inline packages/patternfly-4/gatsby-theme-patternfly-org/templates/mdx.js]: Remove this line

File: packages/patternfly-4/gatsby-theme-patternfly-org/gatsby-node.js [hunk 3]
```diff
@@ -182,8 +182,8 @@ exports.createPages = ({ actions, graphql }, pluginOptions) => graphql(`
     }
     const hbsInstance = createHandlebars(result.data.partials.nodes);
     const hiddenTitles = (pluginOptions.hiddenPages || []).map(title => title.toLowerCase());
-    const { showBanner = false } = pluginOptions;
-
+    const { showBanner = false, showGdprBanner = false } = pluginOptions;
+    console.log('showGdprBanner: ', showGdprBanner);
```
Comment (redallen) [inline packages/patternfly-4/gatsby-theme-patternfly-org/gatsby-node.js]: Remove this line

File: packages/patternfly-4/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js (hunk 1 above)
Comment (evwilkin) [reply packages/patternfly-4/gatsby-theme-patternfly-org/layouts/sideNavLayout/sideNavLayout.js]: Done - thanks!

**Pull request merged**